base/process.jl

```diff
@@ -72,6 +72,10 @@ backticks, e.g.
 """
 Cmd
 
+hash(x::Cmd) = hash(x.exec, hash(x.env, hash(x.ignorestatus, hash(x.dir, hash(x.flags)))))
```

Comment:
This should actually implement `hash(x::Cmd, h::UInt)` and pass `h` through as `hash(x.flags, h)`.